### ISSUE TITLE
How to use the Mathematica Test Suite?

### ISSUE THREAD

**Issue by rljacobson** [RuleBasedIntegration/JOSS-Publication#6]
There is some nice documentation for the Mathematica test program:
https://github.com/RuleBasedIntegration/IntegratorTestProgram
But it looks like the program hasn't been committed yet. The format appears to be:
```mma
{ integrand, var, stepsexpected, resultexpected }
```
I think having even one line somewhere saying what those fields are would be minimally sufficient.

**Comment by halirutan** [comment]
I am to blame for this. Albert asked me long ago if I can review his integrator program and adapt it so that we can push it into the repository. Although it's not in a repository, Albert of course shares the program with everyone who wants it and I simply forgot to mention this somewhere.

As a first dirty solution, I have included the meaning of the test-case structure right at the top of the [Integrator Test Program readme](https://github.com/RuleBasedIntegration/IntegratorTestProgram). In addition, I have created [an issue](https://github.com/RuleBasedIntegration/Rubi/issues/17) for this. I would really like to see that some more people jump onboard and help.
